tests/ci/codebuild/common/run_nonroot_target.yml

```diff
@@ -11,11 +11,11 @@ phases:
   install:
     run-as: root
     commands:
-      # Let postgres user in docker image take ownership of codebuild artifacts.
+      # Let non-root user in docker image take ownership of codebuild artifacts.
       - chown -R postgres:postgres /codebuild/output
```

Comment:
What's up with this change? It's still giving ownership to the postgres user.

Reply:
I'm just trying to clarify that postgres is the non-root user here. I tried using changing the user name to a better name, but that'll result in issues where existing scripts on main still expect `postgres`.